common/sgx/iohandler/file_io.cpp

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2019 Intel Corporation
```

Comment:
Any specific reason why this is a new file ? My understanding is existing files trusted_iohandler.cpp/.h will we renamed to file_io.cpp/.h. If this is unrelated to this PR , better to raise a separate PR which can be merged first before this one.

Reply:
this is a custom file io handler file. earlier this file was trusted_iohandler.cpp.

Reply:
Use "git mv" command. This change should ideally be in a separate PR

Reply:
Yes, but as part of this PR i have to make some modification to this file. Hence i renamed the file in this PR only.

Reply:
It's not clear what the modifications are. The entire file contents is being shown as new changes in this PR

Reply:
i will try to delete this file and do "git mv". Hopefully it will be clear.

Reply:
Since there are modifications after file renaming, they are appearing as new files.